src/entities/position.ts

```diff
@@ -282,7 +283,8 @@ class Position {
   
         return this.amm.descale(cashFlow);
 
-      } catch (e) {
+      } catch (error) {
+        Sentry.captureException(error);
```

Comment:
What is the error message in this case? We haven't created a human readable message for some reason in this case, so I am defaulting to the "error" object in hope it will contain enough information for us. If not we can always revisit this piece of code.